### PR TITLE
Layer for richness nos section

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: services/diversity/api_doc/richness.js [hunk 1]
```diff
@@ -77,3 +77,18 @@
...
  */
+
+/**
+ * @apiDefine NOSLayerExample
+ * @apiSuccessExample {binary} Success-Example:
+ * image/png
+ */
+
+/**
+ * @apiDefine NSThresholdsExample
```
Comment (erikasv) [inline services/diversity/api_doc/richness.js:88]: This name is the same as the defined in line 38, I suppose the correct name is `NOSLayerThresholdsExample` ?

File: services/diversity/src/routes/richness.js [hunk 2]
```diff
@@ -153,5 +153,69 @@ module.exports = (errorHandler, Richness) => {
...
+   * @apiParam (Query params) {String|Number} areaId area id
+   * @apiParam (Query params) {String} group to select the proper layer
+   *
+   * @apiSuccess {Object} result
+   * @apiSuccess {String} result.min min value inside the layer
+   * @apiSuccess {Number} result.max max value inside the layer
+   *
+   * @apiExample {curl} Example usage:
+   *  /richness/number-species/layer/thresholds?areaType=ea&areaId=CARDER&group=total
+   * @apiUse NOSLayerThresholdsExample
```
Comment (erikasv) [inline services/diversity/src/routes/richness.js:206]: When generating docs this error is returned: `Referenced groupname does not exist`

File: services/diversity/src/routes/richness.js [hunk 3]
```diff
@@ -153,5 +153,69 @@ module.exports = (errorHandler, Richness) => {
...
+   * @api {get} /richness/number-species/layer NOSLayer
+   * @apiName NOSLayer
+   * @apiVersion 1.0.0
+   * @apiDescription
+   * Layer of a specific group for richness - number of species in a given area. Parameter group
+   * may be selected from: total, endemic, invasive and threatened.
+   *
+   * @apiParam (Query params) {String} areaType area type
+   * @apiParam (Query params) {String|Number} areaId area id
+   * @apiParam (Query params) {String} group to select the proper layer
```
Comment (erikasv) [inline services/diversity/src/routes/richness.js:167]: You should document the options here as you did in the the other endpoint

**Review by erikasv — CHANGES_REQUESTED** [review]

File: services/diversity/api_doc/richness.js (hunk 1 above)
Comment (cazapatamar) [reply services/diversity/api_doc/richness.js]: Yes, fixed!

File: services/diversity/src/routes/richness.js (hunk 2 above)
Comment (cazapatamar) [reply services/diversity/src/routes/richness.js:206]: Fixed!

File: services/diversity/src/routes/richness.js (hunk 3 above)
Comment (cazapatamar) [reply services/diversity/src/routes/richness.js]: Done!

**Review by erikasv — APPROVED** [review]

**Pull request merged**